packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.props.ts

```diff
@@ -14,6 +14,15 @@ export interface ScrollChange {
     end: boolean;
 }
 
+/**
+ * Overflow interface for consumers
+ * Includes all properties found in ScrollChange
+ * 'overflow' is true when there are enough items to cause overflow
+ */
+export interface OverflowChange extends ScrollChange {
```

Comment:
I don't feel a single overflow value is the most intuitive API:

I would prefer to see "overflowStart" and "overflowEnd" values explicitly. With this approach, my code to hide/show flippers might look something like:

```
let showStartFlipper = true;
let showEndFlipper = true;

if (overflowChange.start) {
    showStartFlipper = false;
}

if (overflowChange.end || !overflowChange.overflow) {
    showFlipperEnd = false;
}
```

What is the reason we would land this instead of simply:

```
let showStartFlipper = overflowChange.overflowStart;
let showEndFlipper = overflowChange.overflowEnd;
```

Apologies if I'm being thick - we can sync on a call if that is preferred.

Reply:
Updated per our convo.